pkg/northbound/gnmi/subscribe_test.go

```diff
@@ -421,6 +423,82 @@ func Test_SubscribeLeafStreamDelete(t *testing.T) {
 
 }
 
+// Test_SubscribeLeafStreamWithDeviceLoaded tests subscribing with mode STREAM for an existing device
+// and then issuing a set request with updates for that path
+func Test_SubscribeLeafStreamWithDeviceLoaded(t *testing.T) {
+	server, mgr := setUp()
+	mgr.DeviceStore = &topocache.DeviceStore{
```

Comment:
maybe should ask Ray about gomock to mock this up - I know he has been working with it lately